src/integrations/prefect-dask/prefect_dask/task_runners.py

```diff
@@ -346,13 +346,19 @@ def client(self) -> PrefectDaskClient:
                     # Default to LocalCluster only if no specific class was provided or resolved
                     class_to_instantiate = distributed.LocalCluster
 
-                self.logger.info(
-                    "Creating a new Dask cluster with "
-                    f"`{to_qualified_name(class_to_instantiate)}`"
-                )
-                self._connect_to = self._cluster = self._exit_stack.enter_context(
-                    class_to_instantiate(**self.cluster_kwargs)
-                )
+                cluster_name = to_qualified_name(class_to_instantiate)
+                self.logger.info(f"Creating a new Dask cluster with `{cluster_name}`")
+
+                try:
+                    self._connect_to = self._cluster = self._exit_stack.enter_context(
+                        class_to_instantiate(**self.cluster_kwargs)
+                    )
+                except Exception as e:
+                    self.logger.error(
+                        f"Failed to create {cluster_name} cluster: "
+                        f"{type(e).__name__}: {str(e)}"
```

Comment:
this looks nonstandard to me vs. `self.logger.error(f"Failed to create {cluster_name} cluster: ", exc_info=True)`

Reply:
good call - updated in [c4d1a94](https://github.com/PrefectHQ/prefect/pull/18432/commits/c4d1a9449d438e392ce30e96630812277f59cbff)